packages/object-extend/index.tests.ts

```diff
@@ -51,6 +47,10 @@ extend("");
 extend(false);
 // @ts-expect-error
 extend();
+// @ts-expect-error
+extend([]);
+// @ts-expect-error
+extend(() => {});
```

Comment:
I moved these tests to the "Not OK" zone because it seems that they are not intended usages of the `extend` function.